cmd/broker/bind_create_test.go

```diff
@@ -476,6 +476,42 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		//then
 		require.Equal(t, http.StatusBadRequest, response.StatusCode)
 	})
+
```

Comment:
those test cases does not need 'envtest' (envtest increases the time of the execution). Please move those test cases to binding_test or even to internal/broker  - you need to decide